modules/serverless-typescript-functions-overriding-liveness-readiness.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies
+//
+// * functions/reference/serverless-developing-typescript-functions.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-typescript-functions-overriding-liveness-readiness_{context}"]
+= Overriding liveness and readiness probe values
+
+You can override `liveness` and `readiness` probe values for your TypeScript functions. This allows you to modify the timing of health checks performed on the function.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving are installed on the cluster.
+* You have installed the Knative (`kn`) CLI.
+* You have created a function by using `kn func create`.
+
+.Procedure
+
+. Add the `readinessProbe` and `livenessProbe` sections with new time values to the `container` section of the `func.yaml` file:
```

Comment:
I think for Typescript it is more strainghtforward. What devs needs to do is set the environment variable `LIVENESS_URL` and `READINESS_URL` on the `run.envs` section of `func.yaml` as well as set the endpoint on the `deploy.healthEndpoints` section on `func.yaml` as well.

For example, on `func.yaml`, user can set the below:

```
run:
  envs:
  - name: LIVENESS_URL
    value: /alive
  - name: READINESS_URL
    value: /ready
deploy:
  healthEndpoints:
    liveness: /alive
    readiness: /ready

``` 
Note the environment variable trick as works for NodeJS functions, so you if you like you can add that to NodeJS section as well